### PR TITLE
Add NationalId

### PR DESCRIPTION
**Pull request by JoseOrtiz**
## Description
KYC needs an illustration not present which is National ID

## Checks
- [x] Requires documentation update
- [ ] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [ ] **MINOR** adds functionality in a backwards-compatible manner.
- [x] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

**Review by rahulpsingh — APPROVED** [review]

**Comment by jantocv** [comment]
@JoseOrtiz please could you confirm that this icon is not the same as `details.svg` file? Thanks.

File: src/icons/nationalId.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<svg viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jasonadt) [inline src/icons/nationalId.svg:1]: Please add the missing `class="icon"` attribute to this line.

**Review by jasonadt — CHANGES_REQUESTED** [review]

**Comment by JoseOrtiz** [comment]
> @JoseOrtiz please could you confirm that this icon is not the same as `details.svg` file? Thanks.

Yeah, it's the same, but it's a very bad name for a ID type icon

**Comment by jasonadt** [comment]
> > @JoseOrtiz please could you confirm that this icon is not the same as `details.svg` file? Thanks.
> 
> Yeah, it's the same, but it's a very bad name for a ID type icon

Then let's please make sure keep only one and remove the other one.

**Pull request closed without merging**